fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java

```diff
@@ -212,7 +212,9 @@ private String getSql(final String name, final String type) {
     }
 
     @Override
-    public String getReportType(final String reportName, final boolean isSelfServiceUserReport) {
+    public String getReportType(final String reportName, final boolean isSelfServiceUserReport, final boolean isParameterType) {
+        if (isParameterType) return "Table";
```

Comment:
Format violations
`if` construct must use {}'s.

Reply:
Thanks - have fixed... strange that spotlessCheck did not complain, we obviously have two different sets of formatting rules in place. Given you've confirmed this works for you, I'll merge this if it passes the automatic tests.